MapboxNavigation/RouteVoiceController.swift

```diff
@@ -82,6 +82,8 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
     var volumeToken: NSKeyValueObservation?
     var muteToken: NSKeyValueObservation?
     
+    fileprivate let categoryOptions: AVAudioSessionCategoryOptions = [.duckOthers, .interruptSpokenAudioAndMixWithOthers]
```

Comment:
Note that this is a read-only instance property, not merely a static constant. Consider turning this symbol into either a static constant or moving it into the file scope.